src/core/models/imagem/imagem.model.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const { mongoose } = require('../../database');
+
+const imagemSchema = mongoose.Schema({
+  nome: String,
+  data: String,
```

Comment:
qual a diferença de data para dataCriação? se data for de dados, não deveria ser do tipo Buffer?

Reply:
Concordo,vou mudar quando resolver os testes